pennylane/io/io.py

```diff
@@ -843,14 +843,14 @@ def from_quil_file(quil_filename: str):
     return plugin_converter(quil_filename)
 
 
-def from_qasm3(quantum_circuit: str, wire_map: dict = None):
+def from_qasm3(quantum_circuit: str, wire_map: dict = None, **kwargs):
```

Comment:
I'd suggest moving `**kwargs` to the returned function. 

So it'd look something like:
```
from_qasm3(program)(a = 2)
```